src/utils/delegation.ts

```diff
@@ -26,7 +26,7 @@ export default async function getDelegatesBySpace(
 
   let pivot = 0;
   const result = new Map<string, Delegation>();
-  const spaceIn = buildSpaceIn(space);
+  const spaceIn = space ? buildSpaceIn(space) : null;
```

Comment:
It takes too much time to get all delegations 😅 (100.418 seconds) but yea works

Reply:
Yeah, this will be reused on the server to only do it once every 24 hours for all spaces on given network so it shouldn't be terrible.